apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -196,7 +196,7 @@ export default function ListBox({
     if (typeof setCount === 'function' && layout) {
       setCount(layout.qListObject.qSize.qcy);
     }
-  }, [layout]);
+  }, [layout, layout && layout.qListObject.qSize.qcy]);
```

Comment:
Is it safe to chain this? Looks succeptible to "Cannot read properties of 'undefined'"

Reply:
I would say so since we are checking for layout, manually tested it and looks good.

Reply:
It seems to work fine, yes.
I would have preferred to write it using optional chaining, like: `layout?.qListObject...` but it's not allowed in nebula at the moment.

Reply:
The ugly long version of the above would be
`layout && layout.qListObject && layout.qListObject.qSize && layout.qListObject.qSize.qcy`

Up to you 🙈

Reply:
We are already assuming that `qListObject.qSize.qcy` exist once layout is in place (line 197) today, so this adds no extra risk the way I see it.